ballerina/tests/test_client.bal

```diff
@@ -0,0 +1,693 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/os;
+
+configurable string clientId = os:getEnv("CLIENT_ID");
+configurable string clientSecret = os:getEnv("CLIENT_SECRET");
+configurable string refreshToken = os:getEnv("REFRESH_TOKEN");
+configurable string refreshUrl = os:getEnv("REFRESH_URL");
+
+ConnectionConfig config = {
+    auth: {
+        clientId: clientId,
+        clientSecret: clientSecret,
+        refreshToken: refreshToken,
+        refreshUrl: refreshUrl   
+    }
+};
+
+@test:Config {}
+function testCreateAndDeleteCalendar() returns Error? {
+    Client client1 = check new(config);
+    string summary = "Test Calendar 1";
+    Calendar cal = {
+        summary: summary
+    };
+    Calendar unionResult = check client1->/calendars.post(cal);
+    test:assertEquals(unionResult.summary, summary);
+
+    Error? res = check client1->/calendars/[<string>unionResult.id].delete();
+    test:assertEquals(res, ());
+}
+
+@test:Config{}
+function testGetCalendar() returns Error?  {
+    Client client1 = check new(config);
+    string summary = "Test Calendar";
+    Calendar cal = {
+        summary: summary
+    };
+    Calendar createdCal = check client1->/calendars.post(cal);
+    Calendar retrievedCal = check client1->/calendars/[<string>createdCal.id].get();
+    test:assertEquals(retrievedCal.summary, summary);
+    Error? res = check client1->/calendars/[<string>createdCal.id].delete();
+    test:assertEquals(res, ());
+}
+
+@test:Config{}
+function testUpdateCalendar() returns Error?  {
+    Client client1 = check new(config);
+    string summary = "Test Calendar";
+    Calendar cal = {
+        summary: summary
+    };
+    Calendar createdCal = check client1->/calendars.post(cal);
+    string newSummary = "Updated Test Calendar";
+    createdCal.summary = newSummary;
+    Calendar updatedCal = check client1->/calendars/[<string>createdCal.id].put(createdCal);
+    test:assertEquals(updatedCal.summary, newSummary);
+    Error? res = check client1->/calendars/[<string>createdCal.id].delete();
+    test:assertEquals(res, ());

```

Comment:
```suggestion
    check client1->/calendars/[<string>createdCal.id].delete();
```

Reply:
Fixed in [5ec2e48](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/134/commits/5ec2e48c2cbc1fa15e119f5cee8d8021de16eae5)